normandy/recipes/api/serializers.py

```diff
@@ -102,6 +103,43 @@ class Meta:
             'is_approved',
         ]
 
+    def validate_arguments(self, value):
```

Comment:
@rehandalal Added some serializer tests with a bunch of questions for you. I'll also add some api tests to make sure the right responses are returned